### DISCUSSION TITLE
2024.03.19

### DISCUSSION THREAD

**Discussion by DCSBL** [homewizard/api-documentation#74] [Announcements]
# Summary
The features 'Identity' and 'system' are now available in the newest firmware version. This version is currently in beta and is tested by a small group of users. The firmware will be released to all users when the beta phase is completed.

## What's Changed
* Add support for 'identify' and 'system' for HWE-WTR by @DCSBL in https://github.com/homewizard/api-documentation/pull/72
* Adjust headers and model numbers to make style more consistent by @DCSBL in https://github.com/homewizard/api-documentation/pull/73

## Dependencies
* Bump follow-redirects from 1.15.5 to 1.15.6 by @dependabot in https://github.com/homewizard/api-documentation/pull/71

**Full Changelog**: https://github.com/homewizard/api-documentation/compare/2024.03.12...2024.03.19

<hr /><em>This discussion was created from the release <a href='https://github.com/homewizard/api-documentation/releases/tag/2024.03.19'>2024.03.19</a>.</em>

**Comment by RaymiiOrg** [comment]
Can my watermeter be included in the test? I'd love to disable cloud and get rid of the ugly red ring it now has because i blocked internet access.

**Reply by DCSBL** [reply]
Yes, you can request beta-access via this form; https://forms.gle/ViUWaBBjjdQunquK9. 

You will receive an e-mail within a day or two, then bring your Watermeter online and select 'Check for update' in the device settings (HomeWizard app).

**Reply by RaymiiOrg** [reply]
Thank you!
